src/Microsoft.AspNetCore.Server.IIS/HttpProtocol.FeatureCollection.cs

```diff
@@ -0,0 +1,270 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Taking from Kestrel. We will see if these features can be combined in some way in the future.